### PR TITLE
fix: enable scrolling when hovering components in IE11

### PR DESCRIPTION
**Pull request by pmpinto**
Hey @luke-m!

Gulp is doing these kebab-case to camelCase changes. Not sure why or if it's safe...

The relevant part of this PR is to include in the IE11 code block in the dist files. This change was originally pushed by @ctmm in #4 and was not usable since it was not in dist.

### CODE REVIEW

File: dist/fullpage.js [hunk 1]
```diff
@@ -2809,7 +2809,7 @@
 
             removeClass(element, NO_TRANSITION);
             return css(element, {
-                '-webkit-transition': transition,
+                'webkitTransition': transition,
```
Comment (clemens-vi) [inline dist/fullpage.js]: this change looks like it's removing a vendor prefix here !? :
'-webkit-transition'

Does not feel a "safe" change to me...

File: dist/fullpage.js [hunk 2]
```diff
@@ -3438,8 +3446,8 @@
             css(container, {
                 'height': '',
                 'position': '',
-                '-ms-touch-action': '',
-                'touch-action': ''
+                'msTouchAction': '',
```
Comment (clemens-vi) [inline dist/fullpage.js]: also here: looks like this removes the vendor prefix.

**Comment by clemens-vi** [comment]
> Gulp is doing these kebab-case to camelCase changes. Not sure why or if it's safe...

this definitley does not feel safe...

**Review by clemens-vi — CHANGES_REQUESTED** [review]
> Gulp is doing these kebab-case to camelCase changes. Not sure why or if it's safe...

this definitley does not feel safe... Could you either find out if it really still works as intended, or revert those changes somehow?

**Comment by luke-m** [comment]
>Gulp is doing these kebab-case to camelCase changes. Not sure why or if it's safe...

I don't think gulp is doing this. I just tried it and it's not happening for me. I think it's your linter. Can you please check that?  
I ran into the same problem when I was working in this repo and it's definitely not safe to push like this.

**Review by luke-m — CHANGES_REQUESTED** [review]
See my comment above.

**Comment by pmpinto** [comment]
@clemens-vi @luke-m 
Thanks for pointing this out.
Indeed, it wasn't Gulp as I initially thought. I narrowed it down to the stylelint+ VSCode extension that was formatting JavaScript files on save as well (for those files that contain styles in JS).

Temporarily disabled it, and pushed only the relevant changes this time.

Please take another look.

**Review by luke-m — APPROVED** [review]

**Pull request merged**